tools/tune/multiband_drc/multiband_drc_build_blob.m

```diff
@@ -1,12 +1,16 @@
 function blob8 = multiband_drc_build_blob(num_bands, enable_emp_deemp,
 					  emp_coefs, deemp_coefs,
 					  crossover_coefs, drc_coefs,
-					  endian);
+					  endian, ipc_ver)
 
 if nargin < 7
         endian = 'little'
 endif
 
+if nargin < 8
```

Comment:
Since the number of input arguments has increased to eight, It is not possible to improve the logic under the if elseif implementation.

Please also consider removing `endif` to `end`.

There's a `endfunction`, at line number 82, please consider replacing  `endfunction` to `end`

Reply:
Yep, fixed.